thunder/__init__.py

```diff
@@ -746,6 +746,9 @@ def wrapped(*args, **kwargs):
 
         return wrapped
 
+    # For more context see `NOTE: Split autograd.Function`
+    disable_split_autograd: bool = compile_options.get("disable_split_autograd", False)
```

Comment:
Can you please make this specific to thunderfx or at least name these to make it clear that they're only supposed to be used with thunderfx?